Taskfile.yml

```diff
@@ -0,0 +1,55 @@
+---
+version: '3'
+vars:
+  SEMVER: 'v0.0.0'
+
+tasks:
+  build:
+    desc: Build go binary
+    cmds:
+      # For Matrix build a Taskfile v3.39.0 or newer is needed
+      - for:
+          matrix:
+            OS: ['linux']
+            ARCH: ['amd64', 'arm', 'arm64']
+        cmd: GOOS="{{.ITEM.OS}}" GOARCH="{{.ITEM.ARCH}}" go build -ldflags="-s -w" -o "bin/bmc-test-go-{{.ITEM.OS}}-{{.ITEM.ARCH}}-{{.SEMVER}}"
```

Comment:
We would need to still have the 

        ln -s -f bmc-test-armv5 bmc-test-cross

since thats what fwci expects but we can add that back later on.

Reply:
If renaming is needed, it should not be done during build ;)

If fwci needs specific name, the renaming should be part of `deploy` or `update fwci` or somethings like that.